### PR TITLE
Mnaveau/solo 12

### PR DESCRIPTION
**Pull request by MaximilienNaveau**
## What changed?

Create a dgm_parameters_solo8 and a dgm_parameters_solo12.

## How did I tested it:

To be tested:
- in simulation
- in the real robot

## To be merged after:

- [ ] Another PR in dg_blmc_robots which is not up yet...

### CODE REVIEW

File: config/dgm_parameters_solo8.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # 
-# @file dgm_parameters.yaml
+# @file dgm_parameters_solo8.yaml
```
Comment (jviereck) [inline config/dgm_parameters_solo8.yaml:2]: You have to update the `Solo8Config` and `Solo12Config` `yaml_path` as well.

https://github.com/open-dynamic-robot-initiative/robot_properties_solo/blob/master/python/robot_properties_solo/config.py#L69
https://github.com/open-dynamic-robot-initiative/robot_properties_solo/blob/master/python/robot_properties_solo/config.py#L129

**Review by jviereck — CHANGES_REQUESTED** [review]

File: config/dgm_parameters_solo12.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      size: 12
+    joint_target_torques:
+      size: 12
+    joint_encoder_index:
+      size: 12
+    # additional data
+    contact_sensors:
+      size: 4
+    slider_positions:
+      size: 2
```
Comment (jviereck) [inline config/dgm_parameters_solo12.yaml]: For compatibility with `solo8`, please use `size: 4` for the sliders here.

**Review by jviereck — CHANGES_REQUESTED** [review]

**Comment by MaximilienNaveau** [comment]
I fixed this python/config.py after a rebase and force push.

**Review by jviereck — APPROVED** [review]

**Comment by MaximilienNaveau** [comment]
Thanks @jviereck for checking the PR, I would not merge it right away as it would break dg_blmc_robots. The PR is gonna come once I double checked that the DGM works on Solo12.

**Pull request merged**